### PR TITLE
overlay2: fix unmounting layer without merge dir

### PR DESCRIPTION
**Pull request by tonistiigi**
@dmcgowan 

Signed-off-by: Tonis Tiigi <tonistiigi@gmail.com>

### CODE REVIEW

**Review by cpuguy83 — APPROVED** [review]
Looks ok but why wouldn't there be a merge dir?

File: daemon/graphdriver/overlay2/overlay.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/docker/docker/pkg/mount"
 	"github.com/docker/docker/pkg/parsers"
 	"github.com/docker/docker/pkg/parsers/kernel"
-	"github.com/docker/go-units"
+	units "github.com/docker/go-units"
```
Comment (cpuguy83) [inline daemon/graphdriver/overlay2/overlay.go]: looks like it's not needed?

**Review by dmcgowan — APPROVED** [review]
One of those situations where I would have thought this was already done

LGTM

**Comment by dmcgowan** [comment]
@cpuguy83 https://github.com/docker/docker/pull/31069/files#diff-a15814b493b1d5546c0fd85701e21142L480

File: daemon/graphdriver/overlay2/overlay.go (hunk 1 above)
Comment (vdemeester) [inline daemon/graphdriver/overlay2/overlay.go]: I do like that though 👼

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Comment by thaJeztah** [comment]
cherry-picked into 17.03.1 through https://github.com/docker/docker/pull/31754

**Pull request merged**